spec/diagnose_spec.rb

```diff
@@ -620,7 +619,8 @@
             "env" => "development"
           },
           "system" => {
-            "active" => true
+            "active" => true,
+            "send_session_data" => true
```

Comment:
How come it shows up as `system` and not `initial`? Is it because it's derived from `skip_session_data`?

If the user were to set `skip_session_data: true` in their config file, would it have its own entry on the diagnose test under `file`? Or would it show up as `send_session_data: false` under `system`?

Reply:
> If the user were to set skip_session_data: true in their config file, would it have its own entry on the diagnose test under file? Or would it show up as send_session_data: false under system? 

Both. The user sets `skip_session_data: true` in the file soure, which triggers `send_session_data: false` to be set in the system source.

Reply:
That seems like the right behaviour to me 👍